calendar-helper/index.js

```diff
@@ -174,7 +208,9 @@ class CalendarHelper {
       } else {
         currentRow[weekDay] = Markup.button.callback(
           d.toString(),
-          'calendar-telegram-date-' + CalendarHelper.toYyyymmdd(date)
+          'calendar-telegram-date-' +
+            CalendarHelper.toYyyymmdd(date) +
+            this.options.shortcutButtonCallbackSuffix
```

Comment:
Why aren't we using `shouldPrefixShortcutButtonCallbacks` here?

Reply:
Fixed

Reply:
Sorry, this was my bad. This needs to be always prefixed, otherwise this action will  not work when `shouldPrefixShortcutButtonCallbacks` is false:
https://github.com/gianlucaparadise/telegraf-calendar-telegram/blob/da886ab3b9242b5a1789346c3ab04c80d334e6f9/index.js#L30